python/ray/tests/test_runtime_env_working_dir_remote_uri_2.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+import sys
+
+import pytest
+from pytest_lazyfixture import lazy_fixture
+
+import ray
+
+# This test requires you have AWS credentials set up (any AWS credentials will
+# do, this test only accesses a public bucket).
+
+# This package contains a subdirectory called `test_module`.
+# Calling `test_module.one()` should return `2`.
+# If you find that confusing, take it up with @jiaodong...
+HTTPS_PACKAGE_URI = "https://github.com/shrekris-anyscale/test_module/archive/HEAD.zip"
+S3_PACKAGE_URI = "s3://runtime-env-test/test_runtime_env.zip"
+GS_PACKAGE_URI = "gs://public-runtime-env-test/test_module.zip"
+REMOTE_URIS = [HTTPS_PACKAGE_URI, S3_PACKAGE_URI]
```

Comment:
I guess this part is copied in several files now, should we put it in a single file somewhere?